app/assets/sass/tai19032019.scss

```diff
@@ -167,4 +167,8 @@ hr {
   bottom: 0;
   right: 0;
   z-index: 9999;
-}
\ No newline at end of file
+}
+
+.incomeSourceSummary .heading-48 {
+    margin-bottom: .625em;
```

Comment:
What font-size does the resolve to? Wouldn't it be better to change the class used to another heading size?

Reply:
Updated as per suggestion